lib/gears/object.lua

```diff
@@ -112,6 +127,21 @@ function object:weak_connect_signal(name, func)
     sig.weak[func] = make_the_gc_obey(func)
 end
 
+--- Check if the callback is weakly connected to the signal.
+--
+-- @tparam string name The name of the signal.
+-- @tparam function func The callback to check if connected.
+-- @treturn boolean Whether the signal is connected.
+-- @method is_weak_signal_connected
+function object:is_weak_signal_connected(name, func)
```

Comment:
could we merge both strong and weak variants into one? I don't really see an use case where it would make a difference.

Reply:
done